data-validation/multiple-coordinate-schemas/README.md

```diff
@@ -172,13 +180,21 @@ The following policy tests if incoming messages match either the USA schema or t
           "level": "WARN",
           "message": "The client ${clientId} attempted to publish invalid coordinate data: ${validationResult}"
         }
+      },
+      {
+        "id": "incrementBadCoordinatesMetric",
+        "functionId": "Metrics.Counter.increment",
+        "arguments": {
+          "metricName": "invalid-coordinates",
+          "incrementBy": 1
+        }
       }
     ]
   }
 }
 ```
 
-When validation succeeds it will be logged at the `INFO` level and include the client ID using the `${clientId}` string substitution. When validation fails it will be logged at the `WARN` level and also include the reason for failure using the `${validationResult}` string substitution.
+When validation succeeds it will be logged at the `INFO` level and include the client ID using the `${clientId}` string substitution. When validation fails it will be logged at the `WARN` level and also include the reason for failure using the `${validationResult}` string substitution. Moreover, for both cases custom metrics are created: `valid-coordinates` and `invalid-coordinates`.
```

Comment:
```suggestion
When validation succeeds it will be logged at the `INFO` level and include the client ID using the `${clientId}` string substitution. When validation fails it will be logged at the `WARN` level and also include the reason for failure using the `${validationResult}` string substitution. Moreover, for both cases custom metrics are created: `com.hivemq.data-governance-hub.data-validation.custom.counters.valid-coordinates` and `com.hivemq.data-governance-hub.data-validation.custom.counters.invalid-coordinates`.
```